### PR TITLE
Update RD policy with examples and video submission guidelines

### PR DESCRIPTION
**Pull request by chasballew**
@krallin @fancyremarker

### CODE REVIEW

File: content/legal/responsible_disclosure.md [hunk 1]
```diff
@@ -51,17 +49,29 @@ Only the following services are in-scope:
...
 - Presence of application or web browser "autocomplete" or "save password" permission
 
+#### Example Issues
+The following types of issues are examples of reports we have paid cash bounties for:
+- Arbitrary directory traversal on Git endpoints
+- Improper direct object references
+- HTML injection
+
+#### Submissions
+Please email [security@aptible.com](mailto:security@aptible.com) to report an issue. If you would like to include a video, please link to an unpublished YouTube URL. We cannot accept binary videos or files.
```
Comment (krallin) [inline content/legal/responsible_disclosure.md]: I'd be a little more explicit and state that we will not accept binary attachments of any kind, including videos.

Comment (krallin) [inline content/legal/responsible_disclosure.md]: I would also ask that people don't attach PDFs. The same problems are present there as with videos.

Comment (chasballew) [inline content/legal/responsible_disclosure.md]: Fixed in 7c3caa8

Comment (krallin) [inline content/legal/responsible_disclosure.md]: Thanks!

**Pull request merged**